training/train.py

```diff
@@ -275,32 +275,26 @@ def save_hyperparams(s3_client, content_json):
 
 def save_dictionaries(s3_client, package_id_dict, manifest_id_dict):
     """Save the ditionaries for scoring."""
-    HPF_package_id_dict = os.path.join("maven", DEPLOYMENT_PREFIX,
-                                       MODEL_VERSION, "trained-model/package_id_dict.json")
-    HPF_manifest_id_dict = os.path.join("maven", DEPLOYMENT_PREFIX,
-                                        MODEL_VERSION, "trained-model/manifest_id_dict.json")
-    if not ((s3_client.object_exists(HPF_package_id_dict)) and
-            (s3_client.object_exists(HPF_manifest_id_dict))):
-        pkg_status = s3_client.write_json_file(
```

Comment:
What if, we retrain the model on the same configuration, then It will replace the existing files right?

Reply:
Yes.

Reply:
Ok, thanks!